packages/opentelemetry-plugin-grpc/src/grpc.ts

```diff
@@ -135,15 +134,13 @@ export class GrpcPlugin extends BasePlugin<grpc> {
       plugin._logger.debug('patched gRPC server');
 
       return function register<RequestType, ResponseType>(
-        // tslint:disable-next-line:no-any
```

Comment:
Would like to enable the no-any rule, even if we have to put ignores in some places. I try to avoid the use of any wherever possible.

Reply:
When you lint your code, you have warnings... and it shows lines, files and the rule you don't respect. It's less noisy.